src/tests/ftest/control/daos_admin_privileged.yaml

```diff
@@ -13,3 +13,11 @@ server_config:
     # bdev_list: ["aaaa:aa:aa.a","bbbb:bb:bb.b"]
     scm_class: dcpm
     scm_list: ["/dev/pmem0"]
+  transport_config:
+    allow_insecure: True
+agent_config:
+  transport_config:
+    allow_insecure: True
+dmg:
+  transport_config:
+    allow_insecure: True
```

Comment:
Why are these tests allowed to run in insecure mode? The commit message does not include any information about why some tests have it enabled and some do not.